### PR TITLE
Update readme.org

### PR DESCRIPTION
**Pull request by freizl**

### CODE REVIEW

File: readme.org [hunk 1]
```diff
@@ -17,7 +17,7 @@ SELECT ARRAY(SELECT UNNEST(ARRAY['id', 'organization_id', 'locked_by', 'lock_rea
   #+begin_example
  {created_at,id,locked_by,lock_note,lock_reasons,lock_source,organization_id,sent_notification_emails}
   #+end_example
-
+change 1
```
Comment (freizl) [inline readme.org:20]: this is my first changes

File: readme.org [hunk 2]
```diff
@@ -48,6 +50,6 @@ I did POC (see details in ~icu_string_comparison.c~) and I got following result,
 |--------------------------------------------------------------------+--------|
 | ~SELECT 'lock_note' COLLATE "en_US" < 'locked_by' COLLATE "en_US"~ | False  |
 | ~icu_string_comparison.c~: ~'lock_note'~ < ~'locked_by'~           | True   |
+| test      |false|
```
Comment (freizl) [inline readme.org:53]: Add another entry to the table

File: readme.org (hunk 1 above)
Comment (hw202207) [reply readme.org:20]: Looks nice

File: readme.org (hunk 2 above)
Comment (hw202207) [inline readme.org:53]: Clean up space

File: readme.org [hunk 3]
```diff
@@ -33,6 +33,8 @@ SELECT 'lock_note' COLLATE "en_US" < 'locked_by' COLLATE "en_US";
 
 I did some google search and the general impression I got is when locale is enabled, sorting in postgres leveraging locale rule from Operating System and the rules may vary cross OS, which could explain why aforementioned queries yields inconsistent result cross OS.
 
+another changes
```
Comment (hw202207) [inline readme.org:36]: Remove this seems redundant

**Review by hw202207 — CHANGES_REQUESTED** [review]
Left some comment

**Review by hw202207 — COMMENTED** [review]
Another feedback

**Pull request closed without merging**